src/routers/admin.py

```diff
@@ -21,12 +22,12 @@ def get_me(request: Request):
 @router.put("/me/", status_code=HTTPStatus.ACCEPTED, dependencies=[Depends(check_domain)])
 @requires("manage-account")
 def update_me(request: Request, user: UserProfile):
-    oauth2_admin.update_user(request.user.id, user.to_keycloak_user())
+    OAuth2Admin.update_user(request.user.id, user.to_keycloak_user())
```

Comment:
you should be creating an instance of the class, not use the class methods in a static fashion, not quite sure how this is working with the `_admin` attribute not constructed within the instance...